sdcm/cluster.py

```diff
@@ -1457,6 +1462,11 @@ def decode_backtrace(self):
                     event.publish()
 
             if self.termination_event.isSet() and Setup.DECODING_QUEUE.empty():
+                self.log.debug("Scylla debug file", scylla_debug_file)
```

Comment:
Why wait until end of test to remove it ?

Once it copied to the monitor, we can remove from the worker, no ?

Reply:
Remove right after copying to monitor node.